internal/secret/secret.go

```diff
@@ -5,10 +5,9 @@ import (
 	"fmt"
```

Comment:
secret package is really around hxkey management. The secret models is separate.